presto-main/src/test/java/io/prestosql/execution/TestSqlTaskExecution.java

```diff
@@ -112,6 +112,7 @@
     private static final OutputBufferId OUTPUT_BUFFER_ID = new OutputBufferId(0);
     private static final CatalogName CONNECTOR_ID = new CatalogName("test");
     private static final Duration ASSERT_WAIT_TIMEOUT = new Duration(1, HOURS);
+    public static final TaskId TASK_ID = new TaskId("query", 0, 0);
```

Comment:
Does this reduce exception stacks printed during tests? or is it a separate change?

Reply:
It does... The tests print some message when cleaning up about an invalid query ID.